Symfony/CS/Tests/Tokenizer/TokensAnalyzerTest.php

```diff
@@ -38,8 +38,14 @@ public function bar4()
 
         return " ({$a})";
     }
+
     public function bar5($data)
     {
+        $message = $data;
+        $example = function ($arg) use ($message) {
+            echo $arg . ' ' . $message;
+        };
+        $example('hello');
```

Comment:
Yes, this should go indeed into 1.11 (even if this will be in different file).
And other code grooming like changing `->isGivenKind(T_WHITESPACE)` into `->isWhitespace()` would be great to be done at 1.x ;)